traitlets/config/tests/test_application.py

```diff
@@ -66,7 +66,7 @@ class MyApp(Application):
 
     aliases = Dict({
                     ('fooi', 'i') : 'Foo.i',
-                    ('j', 'fooj') : 'Foo.j',
+                    ('j', 'fooj') : ('Foo.j', "`j` terse help msg"),
```

Comment:
The point of the help string on traitlets is to specify this help string. Under what circumstances would one set a help string that one didn't want to use?